Notes on renamed pickers (#6317)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker.md

`Returns: IEnumerable<IPublishedContent>`

> The Content Picker allows you to configure the type of content tree to render and what part of it should be rendered. For content it allows you to select a dynamic root node based on the current document using the Content Picker.

Reviewer: The first statement is slightly confusing so tried to suggest one.

{% hint style="info" %}

> The Content Picker is what was known as the **Multinode Treepicker** in version 13 and below.

Reviewer: ```suggestion
The Content Picker was formerly known as the **Multinode Treepicker** in version 13 and below.
```
